templates/docs/community.html

```diff
@@ -0,0 +1,64 @@
+{% extends "skel.html" %}
```

Comment:
Why always the base template? You're not getting a `<title>` this way ;)

This should probably be `section.html`.